lms/djangoapps/courseware/tests/test_access.py

```diff
@@ -44,9 +56,96 @@
     seed_milestone_relationship_types,
 )
 
+from lms.djangoapps.ccx.models import CustomCourseForEdX
+
 # pylint: disable=protected-access
 
 
+class CoachAccessTestCaseCCX(SharedModuleStoreTestCase, LoginEnrollmentTestCase):
```

Comment:
@pdpinch 
I do not know. But I am sure we do not have access tests for ccx in coursware.
2nd thing is that I just incorporate coach role to courseware access to allow him to see progress of individual like staff.

**Should I create this PR on upstream for Ormsbee input?